### PR TITLE
Create a function for printing right-aligned text

### PR DESCRIPTION
**Pull request by illi-kun**

### CODE REVIEW

File: src/output.cpp [hunk 1]
```diff
@@ -354,6 +354,18 @@ void center_print(WINDOW *w, int y, nc_color FG, const char *mes, ...)
     mvwprintz(w, y, x, FG, "%s", text.c_str());
 }
 
+void right_print( WINDOW *w, const int line, const int available_width, const nc_color FG,
```
Comment (codemime) [inline src/output.cpp]: I'd replace `available_width` with a `right_indednt`. It seems more consistent with the "left_print" functions (where `x` is essenitally a left indent) and doesn't enforce both knowing the actual width of the window and calculating a new one. Therefore would be shorter and more expressive in most cases.

Comment (illi-kun) [inline src/output.cpp]: It makes sense, I'll incorporate that.

**Comment by illi-kun** [comment]
done.

**Comment by BevapDin** [comment]
The "\<more\>" and "\<empty\>" strings in the left panel of the sort armor window (shown when the list does not fit the window or is empty) is not printed. I assume it's caused by `print_colored_text` (called from `trim_and_print`, called from the new function). It unconditionally removes *all* tags via `rm_prefix` (removes everything between `<` and `>`), but it should only remove the color tags.

A quick fix would be to change the strings to contain `()` or `[]` or just "empty". The better (optional, maybe separate PR) fix would be to adjust `print_colored_text` to only remove the actually handled color tags.

**Comment by illi-kun** [comment]
I didn't notice that loss but I saw similar issue with '<>' symbols previously in utf8_width(). I'll revert use of right_print for the mentioned instances in this PR and take a look on better handling of `<>` symbols in a separate PR.

**Comment by illi-kun** [comment]
It's ready for test/merge.

**Pull request merged**